x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/options/option_lists.js

```diff
@@ -13,6 +13,7 @@ export const FORMAT_OPTIONS = [
 ];
 
 export const TIMESTAMP_OPTIONS = [
+  'custom',
   'dd/MMM/yyyy:HH:mm:ss XX',
   'EEE MMM dd HH:mm zzz yyyy',
   'EEE MMM dd HH:mm:ss yyyy',
```

Comment:
There are a staggering number of options in this list, many of which are very similar, and now that `custom` is supported maybe we don't need as many because people can tweak one that nearly matches?  What do you think @jgowdy and @peteharverson?

Also, regardless of what's decided for that, all the formats containing `'T'` (lines 99-128) can definitely be deleted from this list because from 7.3 they'll be covered by `ISO8601`.

Reply:
I'm happy to leave the long list of options in the dropdown (minus the ones containing `T` which should be removed), on the basis that it's easier to select from a list of provided options rather than type in your own, plus as you type in characters the list of options narrows.

Reply:
I'll go ahead and remove those options @droberts195 👍

Reply:
Removed in 2adb1002d6ef80fb467803db00d1273545636951